framework/Browser.js

```diff
@@ -0,0 +1,58 @@
+class Browser {
+
+
+  constructor() {
+    this.browser = browser;
+  }
+
+  navigateTo(url) {
+    this.browser.url(url);
+  }
+
+  getCurrentUrl() {
+    return this.browser.getUrl();
+  }
+
+  getPageTitle() {
+    return this.browser.getTitle();
+  }
+
+  findElement(selector) {
```

Comment:
it would be better to add findElements method in this class to have possibility to get multiple elements